proto/workflow.proto

```diff
@@ -169,8 +172,25 @@ message ExecuteWorkflowResponse {
   // The response context.
   context.ResponseContext response_context = 1;
 
+  // TODO(Maggie): Deprecate this field and clean up references
   // The invocation ID of the workflow execution.
   string invocation_id = 2;
+
+  // A list of the actions executed by the API.
+  repeated WorkflowAction actions_started = 3;
+}
+
+message WorkflowAction {
```

Comment:
I feel like `WorkflowAction` is a little ambiguous - it could be referring to the action config, a completed action, or something else. Maybe nesting it under `ExecuteWorkflowResponse` and calling it `ActionStatus` would be less ambiguous?

Sorry if this sounds nit-picky, just want to make sure we avoid confusion down the road and have to introduce a breaking change in the generated client code.

Reply:
Np! Thanks as always for the detailed reviews 🙏